.github/workflows/ci.yml

```diff
@@ -54,10 +54,25 @@ jobs:
       # Install dependencies if cache does not exist
       - name: Install dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-        run: poetry install
+        run: |
+          sed '/zenml/d' pyproject.toml > pyproject2.toml
+          mv pyproject2.toml pyproject.toml
+          poetry lock
```

Comment:
Does `poetry install` not automatically generate a lock file?

Reply:
Our lock file contains zenml as dependency so it has to be updated.